qldbdriver/qldbdriver.go

```diff
@@ -122,25 +121,25 @@ func (driver *QLDBDriver) ExecuteWithRetryPolicy(ctx context.Context, fn func(tx
 			}
 			// Retry
 			retryAttempt++
-			driver.logger.log(fmt.Sprintf("A recoverable error has occurred. Attempting retry #%d.", retryAttempt), LogInfo)
-			driver.logger.log(fmt.Sprintf("Errored Transaction ID: %s. Error cause: %v", txnErr.transactionID, txnErr), LogDebug)
+			driver.logger.logf(LogInfo, "A recoverable error has occurred. Attempting retry #%d.", retryAttempt)
+			driver.logger.logf(LogDebug, "Errored Transaction ID: %s. Error cause: '%v'", txnErr.transactionID, txnErr)
```

Comment:
why '%v' here, but not '%s'? What are the single quotes for?

Reply:
%v can be used for any type and would behave the same as %s for strings.
The single quotes are just to enclose the error message.